Bio/Crystal/__init__.py

```diff
@@ -18,17 +18,21 @@
 
 
 class CrystalError(Exception):
+    """Class to manage errors."""
+
     pass
 
 
 def wrap_line(line):
+    """Add end of line."""
```

Comment:
The point of this function is to insert line breaks for line wrapping at 80characters (presumably a requirement of the file format).

Reply:
I have re-word it, I hope is better now; just to note this module seems incomplete, I haven't found something like 'get_structure' to fill the structure, and the code is from 2002, this might be getting  obsolete.